bucket/windows-application-driver.json

```diff
@@ -6,6 +6,10 @@
     "url": "https://github.com/Microsoft/WinAppDriver/releases/download/v1.2.1/WindowsApplicationDriver_1.2.1.msi",
     "hash": "a76a8f4e44b29bad331acf6b6c248fcc65324f502f28826ad2acd5f3c80857fe",
     "bin": "Windows Application Driver\\WinAppDriver.exe",
+    "post_install": "mv $dir\\System\\* $dir\\Windows*",
+    "env_set": {
+        "APPIUM_WAD_PATH": "$dir\\Windows Application Driver\\WinAppDriver.exe"
+    },
```

Comment:
```suggestion
```
I think users should set it up manually when using it with appium. At least, the envvar`APPIUM_WAD_PATH` have not been created when install via its MSI installer.

Reply:
I will replace with a note.